test/regression/carousel.hermione.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable babel/no-invalid-this */
+'use strict';
```

Comment:
is `'use strict'` needed here?

Reply:
Probably not, it was generated by the gemini -> hermione migration tool